action.yml

```diff
@@ -1,4 +1,4 @@
-name: 'action-php-codesniffer'
+name: 'action-phpcs'
```

Comment:
Is this necessary?

Reply:
changed, as far as I understand its fine to keep the original name.